map_data/geographical_region.txt

```diff
@@ -794,18 +819,96 @@ custom_eastern_kingdoms_jungles = {
 }
 custom_jungles_except_pandaria = {
 	regions = {
-		world_northrend_sholazar_basin
 		custom_eastern_kingdoms_jungles
 		custom_south_seas_except_deserts
+		world_northrend_sholazar_basin
 	}
 }
 custom_jungles = {
 	regions = {
-		custom_jungles_except_pandaria
+		custom_south_seas_except_deserts
+		world_kalimdor_feralas
 		world_pandaria_jade_forest
-		world_pandaria_krasarang_wilds
+		world_northrend_sholazar_basin
+	}
+	duchies = {
+		d_echo_isles
+		d_seething_shore
+		d_glaive_isle
+       		d_lost_isles
+		d_zouchin
+
+		#Krasarang Wilds#
+		d_deepwild
+		d_dojan
+		d_korja
+		d_dawnchaser
+		d_narsong
+		d_nayel
+		d_marista
+		d_zhu
```

Comment:
If you use duchies to make highlighting work, it still doesn't.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114434259-dc381600-9bd3-11eb-8927-4aff2df8012e.png)

</details>